src/utils/util.ts

```diff
@@ -152,7 +159,8 @@ export const compareConfigs = (
 
 export const newCompilerConfig = (
```

Comment:
Just question does it work with contracts that use `remappings`([docs](https://docs.soliditylang.org/en/v0.8.15/using-the-compiler.html?highlight=remappings#input-description)), because we don't send them as I can see.